README.md

```diff
@@ -0,0 +1,55 @@
+<div align="center">
+
+# slurm charms
+
+Manage HPC clusters using [Juju](https://juju.is) and the [SLURM](https://slurm.schedmd.com/overview.html) workload manager.
+
+[![CI](https://github.com/charmed-hpc/slurm-charms/actions/workflows/ci.yaml/badge.svg)](https://github.com/charmed-hpc/slurm-charms/actions/workflows/ci.yaml/badge.svg)
+[![Release](https://github.com/charmed-hpc/slurm-charms/actions/workflows/release.yaml/badge.svg)](https://github.com/charmed-hpc/slurm-charms/actions/workflows/release.yaml/badge.svg)
+[![Matrix](https://img.shields.io/matrix/ubuntu-hpc%3Amatrix.org?logo=matrix&label=ubuntu-hpc)](https://matrix.to/#/#ubuntu-hpc:matrix.org)
+
+</div>
+
+The `slurm-charms` repository provides many of the SLURM daemons that are required to deploy, configure and manage a SLURM-based HPC cluster:
+
+- [`slurmctld`](./charms/slurmctld/): the central management daemon for SLURM.
+- [`slurmd`](./charms/slurmd): the compute node daemon for SLURM.
+- [`slurmdbd`](./charms/slurmdbd): the database daemon for SLURM.
+- [`slurmrestd`](./charms/slurmrestd/): the REST API interface to SLURM.
+
+## Usage
+
+This should be run using Juju 3.x or greater.
+
+#### Deploy a minimal Charmed SLURM cluster
+
+```shell
+$ juju deploy slurmctld --channel edge
+$ juju deploy slurmd --channel edge
+$ juju deploy slurmdbd --channel edge
+$ juju deploy slurmrestd --channel edge
+$ juju deploy mysql --channel 8.0/edge
+$ juju deploy mysql-router slurmdbd-mysql-router --channel dpe/edge
+
+$ juju integrate slurmctld:slurmd slurmd:slurmctld
+$ juju integrate slurmctld:slurmdbd slurmdbd:slurmctld
+$ juju integrate slurmctld:slurmrestd slurmrestd:slurmctld
+$ juju integrate slurmdbd-mysql-router:backend-database mysql:database
+$ juju integrate slurmdbd:database slurmdbd-mysql-router:database
+```
+
+## Project & Community
+
+`slurm-charms` is a project of the [Ubuntu HPC](https://discourse.ubuntu.com/t/high-performance-computing-team/35988)
+community. It is an open source project that is welcome to community involvement, contributions, suggestions, fixes, and
+constructive feedback. Interested in being involved with the development of the project? Check out these links below:
+
+* [Join our online chat](https://matrix.to/#/#ubuntu-hpc:matrix.org)
+* [Contributing guidelines](./CONTRIBUTING.md)
+* [Code of conduct](https://ubuntu.com/community/ethos/code-of-conduct)
+* [File a bug report](https://github.com/charmed-hpc/slurm-charms/issues)
+* [Juju SDK docs](https://juju.is/docs/sdk)
+
+## License
+
+This project is free software, distributed under the Apache Software License, version 2.0. See the [LICENSE](./LICENSE) file for more information.
```

Comment:
We should adapt this blurb from the Slurm snap repository since the charms are under a different license than Slurm itself:

```suggestion
The Slurm charms are free software, distributed under the Apache Software License, version 2.0. See the [Apache-2.0 LICENSE](./LICENSE) file for further details. The Slurm workload manager is licensed under the GNU General Public License, version 2, or any later version. See Slurm's [legal notice](https://slurm.schedmd.com/disclaimer.html) for further licensing information about Slurm.
```